src/components/pathway/SearchPathwayCard.jsx

```diff
@@ -53,6 +53,66 @@ const SearchPathwayCard = ({ hit, isLoading, isSkillQuizResult }) => {
     [isLoading, JSON.stringify(pathway)],
   );
 
+  const loadingCard = () => (
+    <Card
+      className={classNames({ 'skill-quiz-pathway-card': isSkillQuizResult })}
+    >
+      <Card.ImageCap
+        as={Skeleton}
+        duration={0}
+      />
+
+      <Card.Header
+        title={
+          <Skeleton count={2} data-testid="pathway-title-loading" />
+        }
+      />
+
+      <Card.Section>
+        <Skeleton duration={0} data-testid="content-type-loading" />
+      </Card.Section>
+
+    </Card>
+  );
+
+  const searchPathwayCard = () => (
+    <Card
+      isClickable
+      className={classNames({ 'skill-quiz-pathway-card': isSkillQuizResult })}
+    >
+      <Card.ImageCap
+        src={pathway?.cardImageUrl || ''}
+        alt=""
+      />
+
+      <Card.Header
+        title={(
+          <Truncate lines={3} trimWhitespace>
+            {pathway.title}
+          </Truncate>
+        )}
+      />
+
+      <Card.Section classNames="py-3">
+        <div className="d-flex pathway-skill-names">
+          {pathway.skillNames && filterSkillNames(pathway.skillNames).slice(0, MAX_VISIBLE_SKILLS_PATHWAY).map(
+            skillName => (
+              <Badge
+                variant="light"
+                key={skillName}
+                className={classNames(
+                  'pathway-badge d-flex justify-content-center align-items-center mb-2',
+                )}
```

Comment:
nit: unnecessary to use `classNames` if it's just a string (can be passed directly to `className`)